ArduCopter/config.h

```diff
@@ -384,6 +384,19 @@
  # define FLIGHT_MODE_6                  STABILIZE
 #endif
 
+//////////////////////////////////////////////////////////////////////////////
+// SEMI_AUTO_MODE
+//
+#ifndef AUTO_MAN_ALT
+# define AUTO_MAN_ALT   ENABLED
+#endif
+
+//////////////////////////////////////////////////////////////////////////////
+// ENBL_CRS_LOCK
+//
+#ifndef ENBL_CRS_LOCK
+# define ENBL_CRS_LOCK   ENABLED
```

Comment:
Both features should be disabled by default.

Reply:
revised and changed